### PR TITLE
VACMS-8794 Update documentation for running TeamSite pages locally

### PR DESCRIPTION
**Pull request by allisonlu**
## Description
@ddzz helped resolve a recent issue, [#8782](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8782), and flagged that the readme is out of date. Items to update -- all of these need verified, and final changes should be best practices updates, so these exact lines may change:

- Using `yarn`, not `npm`
- `npm watch --` is incorrect
- Steps to run legacy sites locally, per Darius, for verifying:
  - Copy the `headerFooter.json` file to vets-website/build/localhost/generated/. Per @ryguyk, `headerFooter.json` should be generated by running a build on `content-build`.
  - Start `vets-website` with `yarn watch --env local-proxy-rewrite`
  - Navigate to a legacy page as described in the proxy-rewrite README, such as http://localhost:3001/?target=https://www.va.gov/health/

## Original issue(s)
[https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8794](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8794)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/proxy-rewrite/README.md
Comment (ddzz) [inline src/applications/proxy-rewrite/README.md:19]:
```suggestion
yarn run watch --env local-proxy-rewrite
```

Comment (ryguyk) [reply src/applications/proxy-rewrite/README.md:19]: @ddzz Can you shed some light on why you're including `--env` in this command? It doesn't seem strictly necessary to me.

**Review by ddzz — APPROVED** [review]

**Review by it-harrison — APPROVED** [review]

File: src/applications/proxy-rewrite/README.md
Comment (ryguyk) [reply src/applications/proxy-rewrite/README.md]: @ddzz Thanks for the feedback! I _believe_ that this is a slightly different situation than the examples given in the main `README` in that those examples are setting webpack environment variables. In this case, we shouldn't need that. I'm open to feedback from others, but I'd like to update the documentation as accurately as possible if we're going to update it.

This was a change that you helped with a lot, though, Darius, so I'll leave this up to you.  @allisonlu, I'll approve and we can change or update based on Darius's call.

Thanks to you both.

**Review by ryguyk — APPROVED** [review]
See discussion around using `--env`. Approving, but shouldn't merge until that discussion is finalized.

File: src/applications/proxy-rewrite/README.md
Comment (ddzz) [reply src/applications/proxy-rewrite/README.md]: Either way works, so I have no preference here.

**Pull request merged**
